### ISSUE TITLE
`eris chains new myChain` freezes in 0.11.2

### ISSUE THREAD

**Issue by SCBuergel** [eris-ltd/eris-cli#507]
I am running eris on a clean DigitalOcean machine (2GB ram, 2 CPUs, Ubuntu 14.04 x64). I installed docker and eris 0.11.2 from github. Now following the [getting-started guide] (https://docs.erisindustries.com/tutorials/getting-started/) I am running into a show stopper: 

```
$ eris chains new myChain -d
Checking Linux Docker socket                  unix:///var/run/docker.sock
Connecting to Docker                          unix:///var/run/docker.sock
Successfully connected to Docker daemon
Setting up chain                              myChain
```

EDIT: I made a snapshot of the machine after `eris init` and just repeated `eris chains new myChain` on a large machine with 32GB and 8 CPUs just to make sure I am not running into some resource issue but still the marmots seem to be unable to make a chain for me. I did run previous versions of Eris and this command never took long.

NOTE: It is not so straightforward to simply use an older version of Eris due to that Docker version parse issue which has only been resolved in 0.11.2. Therefore, to use an older version of Eris I'd also have to use an older version of Docker. Unfortunately that is not so easy or well documented: I installed [Docker 1.9.1 from binary](https://github.com/docker/docker/releases/tag/v1.9.1) and ended up with `The marmots cannot connect to Docker.` and when doing as requested `sudo services start docker` I get an `sudo: services: command not found`. I guess it should be called `service` (without 's') but then I get `unrecognized service`. End of downgrade attempt.

**Comment by benjaminbollen** [comment]
Hi, thanks for your bug report.

Sorry to ask the "obvious" question first, have you added your user to the docker root group

```(sudo) usermod -a -G docker $USER```
or does it work if you run it as sudo

```sudo eris chains new myChain -d```

**Comment by SCBuergel** [comment]
> Sorry to ask the "obvious" question first, have you added your user to the docker root group
> `(sudo) usermod -a -G docker $USER`
Yes I did add the user to the docker group before trying the reported error. Also running as sudo does not change the problem. 

Let me give a more detailed methods for easier reconstruction (I read on Slack that Dennis was able to reproduce the issue):

1.  started 8GB, 4CPU machine on Digital Ocean running Ubuntu 14.04 x64
2.  followed the Docker installation including prerequisites for 14.04 [as described](https://docs.docker.com/engine/installation/linux/ubuntulinux/):
  - `apt-get update`
  - `apt-get install apt-transport-https ca-certificates`
  - `sudo apt-get update`
  - `sudo apt-get install linux-image-extra-$(uname -r)`
  - `apt-get install apparmor`
3.  installed Docker:
  - `sudo apt-get update`
  - `sudo apt-get install docker-engine`
  - `

**Comment by compleatang** [comment]
Perfect. We have found a bug in how we connect to Docker on 1.10 and @pietv is look at this just now. The bug appears on Docker's side but we are trying to isolate a workaround.

**Comment by pietv** [comment]
Hi @SCBuergel, the fix for this issue is merged with the #develop branch now. Could you please have a go and try if it works for you for your above recipe, when you have time? You can update your branch with the `eris update --branch develop` command. Many thanks.

**Comment by SCBuergel** [comment]
Unfortunately there is some issue: 
```
eris update --branch develop
Error changing directory: %!v(MISSING)
```
That does not tell me or Google a lot. Not sure why I would need that but just in case I was trying with sudo. Had to fight a bit to get the env vars right but it remains the same - note that I am using /root because I am running with default user on Digital Ocean:
```
sudo /bin/sh -c 'GOPATH=/root/work PATH=$PATH:/usr/local/go/bin eris update --branch develop'
Error changing directory: %!v(MISSING)
```

**Comment by pietv** [comment]
Unfortunately this means that you have installed CLI using the binary or package in which case you'd need to wait for update on the #master branch. The error message in this case should be more appropriate (that you don't have `.git` directory).

Glad you have it working now! :)

**Comment by SCBuergel** [comment]
Excellent, this seems resolved in 0.11.3 (currenly running from dev branch).

**Comment by pietv** [comment]
Hi @SCBuergel, thanks very much for your excellent checks and your patience! The fix for this issue is currently on #master branch. You can update your version with the `eris update` command or use the binary distribution from https://github.com/eris-ltd/eris-cli/releases/download/v0.11.2/eris_0.11.2_linux_amd64.tar.gz (Linux). Thanks!
